docs/flank-scripts/command_overview.md

```diff
@@ -0,0 +1,110 @@
+# Flank-scripts command overview
+
+### Command List
+
+- `release` - Group of commands for creating Flank release
```

Comment:
I am wondering if most of the commands in this scope are more related to `github` than the `release`. For example, if we would have similar commands for a different provider that would be more convenient to add a new group for it and reuse the common part like `generate_release_notes`.

Reply:
Decided to make a separate package for `github`